resources/kubernetes/operator/helm/values.yaml

```diff
@@ -805,23 +849,23 @@ collectors:
 
 instrumentation:
```

Comment:
Do we have an  url with general info for instrumentation? Somewhere to explain what this block will do?
Maybe this: https://opentelemetry.io/docs/concepts/instrumentation/zero-code/

Reply:
Definitely, added in https://github.com/elastic/opentelemetry/pull/33/commits/cebe6db6c4fa5139e06985f7ee97caa6a3772666